migrations/add-national_admin-role.js

```diff
@@ -0,0 +1,34 @@
+const _ = require('underscore'),
+      db = require('../db'),
+      async = require('async');
```

Comment:
OK I'll do this because I haven't used it yet… but in my mind I don't actually like this feature. If it helps code be smaller or something that's one thing, but if it's just because people don't want to constantly specify the namespace… I *like* specifying the namespace!